e2e/testcafe-devextreme/tests/scheduler/common/layout/horizontalScroll/horizontalScroll.ts

```diff
@@ -0,0 +1,169 @@
+    endDate: new Date('2021-04-01T22:30:00.000Z'),
+  }, {
+    text: 'Create Icons for Website',
+    startDate: new Date('2021-04-02T17:00:00.000Z'),
+    endDate: new Date('2021-04-02T18:30:00.000Z'),
+  }, {
+    text: 'Upgrade Server Hardware',
+    startDate: new Date('2021-04-02T21:30:00.000Z'),
+    endDate: new Date('2021-04-02T23:00:00.000Z'),
+  }, {
+    text: 'Submit New Website Design',
+    startDate: new Date('2021-04-02T23:30:00.000Z'),
+    endDate: new Date('2021-04-03T01:00:00.000Z'),
+  }, {
+    text: 'Launch New Website',
+    startDate: new Date('2021-04-02T19:20:00.000Z'),
+    endDate: new Date('2021-04-02T21:00:00.000Z'),
+  },
+];
+
+const SCROLLBAR_STYLES = `
+    ::-webkit-scrollbar {
+      -webkit-appearance: none;
+      width: 7px;
+    }
+    ::-webkit-scrollbar-thumb {
+      border-radius: 4px;
+      background-color: rgba(0, 0, 0, .5);
+      -webkit-box-shadow: 0 0 1px rgba(255, 255, 255, .5);
+    }
+    .dx-scheduler-date-table-scrollable .dx-scrollable-container {
+      overflow: scroll !important;
+    }
+`;
+
+const scrollHorizontallyToEnd = ClientFunction(() => {
+  const scheduler = document.querySelector('.dx-scheduler');
+  if (!scheduler) return { success: false, message: 'Scheduler not found' };
+
+  const dateTableScrollable = scheduler.querySelector('.dx-scheduler-date-table-scrollable');
+  if (!dateTableScrollable) return { success: false, message: 'Date table scrollable not found' };
+
+  const scrollableContainer = dateTableScrollable.querySelector('.dx-scrollable-container');
+  if (!scrollableContainer) return { success: false, message: 'Scrollable container not found' };
+
+  const initialScrollLeft = scrollableContainer.scrollLeft;
+  const maxScrollLeft = scrollableContainer.scrollWidth - scrollableContainer.clientWidth;
+
+  scrollableContainer.scrollLeft = maxScrollLeft;
+
+  return {
+    success: true,
+    initialScrollLeft,
+    maxScrollLeft,
+    finalScrollLeft: scrollableContainer.scrollLeft,
+    hasHorizontalScroll: maxScrollLeft > 0,
+  };
+});
+
+test('Scheduler horizontal scroll screenshot test', async (t) => {
+  const {
+    takeScreenshot,
+    compareResults,
+  } = createScreenshotsComparer(t);
+  const scheduler = new Scheduler('#container');
+
+  await insertStylesheetRulesToPage(SCROLLBAR_STYLES);
+
+  await t.wait(100);
+
+  await scheduler.repaint();
+
+  await t.wait(100);
+
+  await t.expect(
+    await takeScreenshot('scheduler-horizontal-scroll-before.png', scheduler.element),
+  ).ok();
+
+  const scrollResult = await scrollHorizontallyToEnd();
+
+  await t
+    .expect(scrollResult.success).ok(`Scroll failed: ${scrollResult.message}`)
+    .expect(scrollResult.hasHorizontalScroll).ok('No horizontal scroll available')
+    .expect(scrollResult.finalScrollLeft)
+    .gt(0, 'Scroll position should be greater than 0');
+
+  await t.wait(500);
+
+  await t.expect(
+    await takeScreenshot('scheduler-horizontal-scroll-after.png', scheduler.element),
```

Comment:
Can we rename screenshot name according to the testing guidelines that Alyar's has created recently on the wiki?